### PR TITLE
Update to CH 1.13.1 to honor supscription status for the remaining valid period after cancellation [stage-2]

### PR DESCRIPTION
**Pull request by settinghead**

### CODE REVIEW

**Comment by settinghead** [comment]
@alex-deaconu @ezequielc or @fjvallarino please review. Thanks.

File: bower.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "angular-ui-codemirror": "~0.3.0",
     "angular-vertilize": "~1.0.1",
     "angular-mocks": "1.5.0",
-    "common-header": "https://github.com/Rise-Vision/common-header.git#v1.13.0",
+    "common-header": "https://github.com/Rise-Vision/common-header.git#1.13.1",
```
Comment (alex-deaconu) [inline bower.json]: @settinghead this works even without the `v`? Thanks.

Comment (settinghead) [inline bower.json]: @alex-deaconu good catch. It's a typo. No idea why the CI passed. Will fix.

Comment (alex-deaconu) [inline bower.json]: @settinghead yeah I was wondering the same thing. From what I can tell, it behaves the same with or without the `v`. Interesting. I guess for consistency we should keep it however.

Thanks for fixing!

Comment (settinghead) [inline bower.json]: Interesting to know!

**Pull request merged**